tests/unit/lms/views/lti/basic_launch_test.py

```diff
@@ -360,10 +380,14 @@ def pyramid_request(self, pyramid_request):
         return pyramid_request
 
     @pytest.fixture
-    def context(self):
+    def context(self, application_instance):
         context = mock.create_autospec(LTILaunchResource, spec_set=True, instance=True)
         context.js_config = mock.create_autospec(JSConfig, spec_set=True, instance=True)
         context.is_canvas = False
+
+        application_instance.check_guid_aligns = mock.Mock()
+        context.application_instance = application_instance
```

Comment:
I used a real application instance here since the settings interface is complex enough that I don't think we want to mock it.